src/cogs/fun.py

```diff
@@ -0,0 +1,143 @@
+import random
+
+import aiohttp
+import discord
+import requests as requests
+from discord.ext import commands
+from datetime import datetime
+
+
+class Fun(commands.Cog):
+    """Fun Bot Commands"""
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name='8ball',
+                      description="Answers a yes/no question.",
+                      brief="Answers from the beyond.",
+                      aliases=['eight_ball', 'eightball', '8-ball'],
+                      pass_context=True)
+    async def eight_ball(self, ctx):
+        possible_responses = [
+            'That is a resounding no',
+            'It is not looking likely',
+            'Too hard to tell',
+            'It is quite possible',
+            'Definitely',
+            'Why don\'t you figure it out yourself you lazy motherfucker',
```

Comment:
Haha a bit too sassy lol? Where'd you get these responses from?

Reply:
https://en.wikipedia.org/wiki/Magic_8-ball#Possible_answers

Reply:
Lol I really just added these for some extra fun. Maybe we could add custom responses in the future.